R/tm_g_response.R

```diff
@@ -195,6 +210,7 @@ tm_g_response <- function(label = "Response Plot",
   )
 }
 
+# UI function for the response module
```

Comment:
@averissimo in other PRs @kartikeyakirar put `@noRd` and `@keywords internal` for those `ui_` and `server_` functions. Can we have them here as well, so it's unified... sorry?

Reply:
Inception... I just [commented](https://github.com/insightsengineering/teal.modules.general/pull/669#discussion_r1502484102) over there saying that we should remove them. Let's decide and we can do this check properly.
I still feel what we have here is proper.

Reply:
Sure thing. It didn't have it before as Kartik's PRs only had this comment.

Reply:
I would not add tags and I would not add comments. The names of the functions and their purpose is self-explanatory. No text/tags at all = no documentation + no indexes.